scripts/lib/CIME/SystemTests/system_tests_common.py

```diff
@@ -82,7 +82,8 @@ def build(self, sharedlib_only=False, model_only=False):
                 start_time = time.time()
                 try:
```

Comment:
Save state of save_build_provenance to be used later in build_indv.

Reply:
Unlike sharedlib, model_only, I can't think of a case where a test type would care if we're saving build provenance or not.